# Generated by Django 5.2.5 on 2025-08-26 22:41

from django.db import migrations, models


class Migration(migrations.Migration):

    dependencies = [
        ('imports', '0002_importrawrecord_error_message_and_more'),
    ]

    operations = [
        migrations.RenameField(
            model_name='importrawrecord',
            old_name='imported_at',
            new_name='product_imported_at',
        ),
        migrations.RenameField(
            model_name='importrawrecord',
            old_name='is_imported',
            new_name='product_is_imported',
        ),
        migrations.AddField(
            model_name='importrawrecord',
            name='price_imported_at',
            field=models.DateTimeField(blank=True, help_text='Timestamp when pricing was imported.', null=True),
        ),
        migrations.AddField(
            model_name='importrawrecord',
            name='price_is_imported',
            field=models.BooleanField(default=False, help_text='True if this record has been successfully imported into price tables.'),
        ),
    ]
